### ISSUE TITLE
On fresh install, ENOENT error for `.cache/functions`

### ISSUE THREAD

**Issue by jlengstorf** [netlify/netlify-plugin-gatsby#41]
**Describe the bug**

On a fresh install of Gatsby, the plugin fails to build due to a missing `.cache/functions` directory.
```
────────────────────────────────────────────────────────────────
  Plugin "@netlify/plugin-gatsby" failed
────────────────────────────────────────────────────────────────

  Error message
  Error: Error message
  ENOENT: no such file or directory, lstat '.cache/functions'
```

**Comment by ascorbic** [comment]
This was fixed in #26, so we need to do a release

**Comment by ascorbic** [comment]
Released in 0.0.2
